features/admin.applications.v1/package.json

```
@@ -39,6 +39,7 @@
         "@wso2is/admin.users.v1": "^2.32.9",
         "@wso2is/admin.userstores.v1": "^2.27.36",
         "@wso2is/admin.wsfed-configuration.v1": "^2.25.205",
+        "@wso2is/common.ui.shared-access.v1": "^0.1.0",
```

Shall we use the workspace protocol?

```suggestion
        "@wso2is/common.ui.shared-access.v1": "workspace:^",
```

---

Fixed with 12101ec308f85c64685f5984555986bc5f6dc015